app/cdap/components/hydrator/components/TopPanel/ActionButtons/index.tsx

```diff
@@ -252,6 +267,17 @@ export const ActionButtons = ({
               <ButtonLabel>Import</ButtonLabel>
             </div>
           </CommonButton>
+          <Label className="import-file-label" for="pipeline-import-config-link">
```

Comment:
Not for this PR, but we should create a file upload component. I started separating logic in #480 but it needs more work and generalization.

Reply:
Yeah I found that one and tried using it. However this toppanel case is weird it needs two onClick handler because of the save modal check requirement inside the angular controller. We'll need to fox that when we do state migration in the end